test/models/profile_test.rb

```diff
@@ -233,6 +233,48 @@ class ProfileTest < ActiveSupport::TestCase
                  Set.new([p8] + profiles)
   end
 
+  context 'fill_from_parent' do
+    NAME = 'Customized profile'
+    DESCRIPTION = 'The best profile ever'
+
+    should 'copy attributes from the parent profile' do
+      profile = Profile.new(parent_profile_id: profiles(:one).id)
+                       .fill_from_parent
+
+      assert_equal profiles(:one).ref_id, profile.ref_id
+      assert_equal profiles(:one).name, profile.name
+      assert_equal profiles(:one).description, profile.description
+      assert_equal profiles(:one).benchmark_id, profile.benchmark_id
+      assert_not profile.external
+    end
+
+    should 'copy rules from the parent profile on save' do
+      profile = Profile.new(parent_profile_id: profiles(:one).id)
+                       .fill_from_parent
+
+      assert_not_equal Set.new(profiles(:one).rules.pluck(:id)),
+                       Set.new(profile.rules.pluck(:id))
+      profile.save
+      assert_equal Set.new(profiles(:one).rules.pluck(:id)),
+                   Set.new(profile.reload.rules.pluck(:id))
+    end
```

Comment:
Ok, @dLobatog, this now copies the rules from the parent profile. I haven't yet implemented tailoring yet - this PR is quite complete as is and already pretty complicated. Perhaps it'd be better to implement tailoring in another PR. For tailoring, I'm thinking we should have a separate API for the profile <-> rules association _and_ allow including relationships in the POST request for profile creation.

1. https://jsonapi.org/format/#crud-creating - notice the `"relationships"` passed in with the POST request
1. https://jsonapi.org/format/#crud-updating-resource-relationships - updating relationships in a PATCH request
1. https://jsonapi.org/format/#crud-updating-relationships - similar thing here, but with just the relationship, not the editable attributes. I think we might hold off on this until V2 and only rely on the PATCH option  for V1

CC @bastilian for your thoughts, as well

Reply:
Just to confirm we are on the same page:

`The relationship’s value will be replaced with the value specified in this member.` 

That means if you send the PATCH request to update the rules relationship, you override it - it's not additive or anything. I think that sounds good to me, but we need to document it, I don't think it's overly surprising but it can be.

Reply:
Right - I think the idea is that you GET a resource, it returns a json document in the schema of a profile, you edit the document with the changes you want, and then PATCH the whole thing back with some thing changed and some things remaining the same.  I guess, one final thing to note is that, we can have default behavior if the relationship isn't provided (assigning parent_profile rules).

For tailoring, you'd GET the parent profile and benchmark - you can use a single request if you `include = benchmark`, copy the rules from that, edit them how you want, and add them to the profile create json before POSTing it to create a new profile with tailored rules.

For documenting, I'll be sure to mention that any provided relationships override all existing values in the relationships param description. Anywhere else we should document it?